clar2wasm/src/tools.rs

```diff
@@ -654,6 +661,43 @@ fn compare_events(events_a: &[EventBatch], events_b: &[EventBatch]) {
     }
 }
 
+pub enum Network {
+    Mainnet,
+    Testnet,
+}
+
+#[derive(Debug)]
+struct ContextConfig {
+    mainnet: bool,
+    chain_id: u32,
+}
+
+static mut CONTEXT_CONFIG: ContextConfig = ContextConfig {
+    mainnet: false,
+    chain_id: CHAIN_ID_TESTNET,
+};
+
```

Comment:
Shouldn't that be a field in `TestEnvironment` instead of a mutable global that is not reset between calls of `crosscheck_with_network`?

Reply:
Yes, makes better sense. Changed.